CHANGELOG.md

```diff
@@ -72,6 +72,7 @@
 * [BUGFIX] Ruler Experimental API: Don't allow rule groups without names or empty rule groups. #3210
 * [BUGFIX] Experimental Alertmanager API: Do not allow empty Alertmanager configurations or bad template filenames to be submitted through the configuration API. #3185
 * [BUGFIX] When using ruler sharding, moving all user rule groups from ruler to a different one and then back could end up with some user groups not being evaluated at all. #3235
+* [BUGFIX] fix timeout handling in `GetObject` method of azure and gcs object clients. #3285
```

Comment:
```suggestion
* [BUGFIX] Honor configured timeout in Azure and GCS object clients. #3285
```

Reply:
I have pushed this change. Thanks for the review!

Reply:
There are now two changelog entries.

Reply:
Could you remove this line, given you added a new one below, please?

Reply:
Sorry, I guess I failed to rebase it properly. Will fix it. Thanks!